cogboard-webapp/src/components/styled/index.js

```diff
@@ -14,6 +14,17 @@ export const WidgetButton = styled(FullWidthButtonOutlined)`
   margin-top: auto;
 `;
 
+export const WidgetIconButton = styled(IconButton)`
+  position: absolute;
```

Comment:
It is just a matter of preference but here we sort css props alphabetically.

Reply:
Ok, done